### ISSUE TITLE
JSONArray.isValidArray return false , JSONArray.parseArray carried out

### ISSUE THREAD

**Issue by KoreHuang** [alibaba/fastjson#2982]
```
String jsonStr="[ { \"activity_type\" : 0, \"activity_id\" : \"***\", \"activity_tip\" : \"***\", \"position\" : \"1\" }, { \"activity_type\" : 0, \"activity_id\" : \"2669\", \"activity_tip\" : \"****\", \"position\" : \"1\" }]";
assertTrue(!JSONArray.isValidArray(jsonStr));
assertTrue(!JSON.isValidArray(jsonStr));
assertTrue(!JSONObject.isValidArray(jsonStr));
assertTrue(JSONArray.parseArray(jsonStr).getString(0) instanceof String);

```

**Comment by KoreHuang** [comment]
<dependency>
      <groupId>com.alibaba</groupId>
      <artifactId>fastjson</artifactId>
      <version>1.2.62</version>
    </dependency>

**Comment by KoreHuang** [comment]
fix issue #2982
